plugin/pick.vim

```diff
@@ -39,6 +39,14 @@ function! PickBuffer()
   call PickCommand(s:BufferListCommand(), "", ":buffer")
 endfunction
 
+function! PickSplitFromBuffers()
```

Comment:
I feel that these names are a bit too descriptive, and doesn't really reflect the commands they execute.

What do you think of renaming them to `PickSplitBuffer` and `PickVerticalSplitBuffer`?